### ISSUE TITLE
Update the PyZabbix Login Wiki

### ISSUE THREAD

**Issue by gcavalcante8808** [lukecyca/pyzabbix#26]
Hello Luke,

I'm using your wonderful library to do some tasks on Zabbix and I found a needed update for a wiki doc page:

https://github.com/lukecyca/pyzabbix/wiki

In my case, my zabbix is using the HTTP/Kerberos Auth, then I Need to use the customized Session token to login. 

However, I cant login using just the information provided in the project's wiki page; the login method need to be called (at least in the Zabbix 2.0) with the user name.

For me, the following code is working like a charm:

https://dpaste.de/rcwp

In other words: if possible, just add the "zapi.login('USER',)" statement and the code will work.

Without this, just the following exception occurs:

Error -32602: Invalid params., Not authorized while sending[...]

The following thread at zabbix foruns pointed me to the way:

https://www.zabbix.com/forum/showthread.php?t=39103

Thank you!!!
